LibTest/io/Directory/watch_A01_t01.dart

```diff
@@ -21,15 +21,14 @@ import "../file_utils.dart";
 main() {
   Directory dir = getTempDirectorySync();
   asyncStart();
-  StreamSubscription s = dir.watch().listen((FileSystemEvent event) {
```

Comment:
there is a lot of duplicated code related to handling of watch stream subscriptions. Consider factoring it out into a helper.